bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/AbstractConfiguration.java

```diff
@@ -215,6 +215,8 @@ public String getMetadataServiceUri() throws ConfigurationException {
                     ledgerManagerType = org.apache.bookkeeper.meta.FlatLedgerManagerFactory.NAME;
                 } else if (factoryClass == LongHierarchicalLedgerManagerFactory.class) {
                     ledgerManagerType = LongHierarchicalLedgerManagerFactory.NAME;
+                } else if (factoryClass == org.apache.bookkeeper.meta.MSLedgerManagerFactory.class) {
```

Comment:
Is anyone still using this?

Reply:
don't think any more is using them. would like to deprecate the whole interface after completing BP-29.

Reply:
:+1: